Tangem/Modules/LegacyTokenList/LegacyTokenListView.swift

```diff
@@ -27,7 +27,7 @@ struct LegacyTokenListView: View {
         .toast(isPresenting: $viewModel.showToast) {
             AlertToast(type: .complete(Colors.Icon.accent), title: Localization.contractAddressCopiedMessage)
         }
-        .searchableCompat(text: $viewModel.enteredSearchText.value)
+        .searchableCompatWithoutCorrection(text: $viewModel.enteredSearchText.value)
```

Comment:
```suggestion
        .searchableCompatWithoutCorrection(text: $viewModel.enteredSearchText.value)
        .autocorrectionDisabled(true)
```

можно же вот так и все